graph/GuidelinesGraph.md

```diff
@@ -256,7 +256,8 @@ If possible, APIs SHOULD use resource-based designs with standard HTTP methods r
 
 |  Microsoft Graph rules for modeling behavior                     |
 |------------------------------------------------------------------|
-| :heavy_check_mark: **MUST** use POST to create new entities in insertable entity sets or collections.<BR>This approach requires the server to produce system generated identities. |
+| :heavy_check_mark: **MUST** use POST to create new entities in insertable entity sets or collections.<BR>This approach requires the server to produce a system generated key. |
+| :heavy_check_mark: **MAY** use PATCH to create new entities in insertable entity sets or collections.<BR>This [Upsert](./patterns/upsert.md) approach requires the caller to provide a key. |
```

Comment:
@OlgaPodo Not sure how to do this one.
It's not really a MUST or a SHOULD, but more of a MAY.  Except we don't have MAY in our legend...

Reply:
two entries don't work together because the first one will always take precedence. Can we express criteria for PATCH better something like 'insertable entity sets or collections with customer provided unique keys'? and something corresponding for the first one? then we can use SHOULD for both use case IMO

Reply:
I don't think we should say both are "should".  I think the POST is a MUST, for consistency.
The Upsert is an additional thing that can be supported.

I've made an update that hopefully works for you?